### PR TITLE
Update settings.py.example

### PR DESCRIPTION
**Pull request by lokeshkvn**
In settings.py.example import for openshift commented

Fix : #40

### CODE REVIEW

File: festcms/settings.py.example [hunk 1]
```diff
@@ -13,8 +13,7 @@ https://docs.djangoproject.com/en/1.8/ref/settings/
 # Build paths inside the project like this: os.path.join(BASE_DIR, ...)
 import os
 
-# import for openshift
-from urllib.parse import urlparse
+# import for openshift from urllib.parse import urlparse
```
Comment (ghoshbishakh) [inline festcms/settings.py.example]: @lokeshkvn This is not proper.

It should be:

```
# import for openshift
# from urllib.parse import urlparse
```

Comment (lokeshkvn) [inline festcms/settings.py.example]: Now commited with the changes you  proposed

**Comment by lokeshkvn** [comment]
Thanks, I will work on it.

**Comment by ghoshbishakh** [comment]
@lokeshkvn This PR is merged! Just remember that for next PR! Thanks.

**Pull request merged**